src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -143,6 +154,7 @@ export default {
 					subline,
 					url: vobject.uri,
 					deletedAt: vobject.dav._props['{http://nextcloud.com/ns}deleted-at'],
+					color: vobject.calendar.color,
```

Comment:
No fallback to calendar color?

Reply:
We don't have a fallback color in the calendar list either. Should I add one?

Reply:
I'd say yes, with https://github.com/nextcloud/calendar/blob/master/src/components/AppNavigation/CalendarList/CalendarListNew.vue#L194 for coherence

Reply:
Thanks for the link. I'll add it.